release-notes/release-notes59.md

```diff
@@ -367,9 +371,7 @@ For information on:
 
 ## Security fixes
 
-{{site.productname}} 5.9 provides fixes for the following security issues:
-
-- changelog
+{{site.productname}} 5.9 provides fixes for the accompanying premium self-hosted server-side components. The `.war` files have been updated to address various security issues.
```

Comment:
Should this include a list of fixes, as this is something we normally include at a high level so customers were aware the a) severity of the security issue and b) roughly what they contained so they can determine the priority. The severity is really the key here, so we at minimum need to say if it's `Critical`, `High`, `Medium` or `Low` severity fixes.

- [ ] Ensure we at least include the severity of the security fixes.

Reply:
This is my fault. I was trying to get out of providing exact CVEs because there were a few, but then there were more in between the releases after the first series of patches. If we need them, I can get them ... it's just a bit more involved because of the numerous patches between the 5.8 and 5.9 release. Some of which only related to things that were introduced after 5.8.

Reply:
I think it's fine if we don't provide the exact list, though that would be preferred, however we should at least mention what was the highest severity issue that was fixed here to give a better indication to customers as to how important it is they upgrade.

Reply:
For example, we could just say something like this (I have no idea if these levels are accurate):
```suggestion
{{site.productname}} 5.9 provides fixes for the accompanying premium self-hosted server-side components. The `.war` files have been updated to address various high and medium severity security issues.
```

Reply:
Looks like high and medium is correct.

Reply:
Thanks!